rotkehlchen/tests/unit/test_savepoints.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+
+import pytest
+
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+
+
+def test_savepoints():
+    conn = GlobalDBHandler().conn
```

Comment:
no no no don't use the global DB like this in tests. If you wanted to use it in tests just use the globaldb fixture.

But again no this is not what I meant.

Skip our global DB completely. There is no need to involve it whatsoever here.

Just do a 
```python
conn = DBConnection(path=':memory:', connection_type=DBConnectionType.GLOBAL, sql_vm_instructions_cb=0)
```

and you have a full DB connection through our driver in memory. (This is pseudocode -- I have not tested it -- may need tweaking)